File: src/engsoft/jogo/patos/Pular_Pata.java
package engsoft.jogo.patos;

public class Pular_Pata implements Padrao_Pular {

	private double altura;

	public Pular_Pata() {
		altura = 2;
	}

	public String pular() {
		return "Pulando como um passaro que pula. Altura: "
				+ getAltura();
	}

	public double getAltura() {
		return altura;
	}

}

File: Pular_Pata.ts
import { Padrao_Pular } from "./Padrao_Pular";

export class Pular_Pata implements Padrao_Pular {
    pular(): string {
        return "Pulo estilo pata!";
    }

    getAltura(): number {
        return 2;
    }
}
